vortexasdk/__init__.py

```diff
@@ -46,3 +46,46 @@
 
 # noinspection PyUnresolvedReferences
 from vortexasdk.check_setup import run_all_checks
+
+# Explicitly define the exported symbols
+__all__ = [
```

Comment:
Following this change should we also update other documentation anywhere else to list the objects available?